src/mudlet-lua/lua/GUIUtils.lua

```diff
@@ -796,8 +795,8 @@ if rex then
         rex.new [[<(?:([0-9]{1,3}),([0-9]{1,3}),([0-9]{1,3}))?(?::(?=>))?(?::([0-9]{1,3}),([0-9]{1,3}),([0-9]{1,3}))?>]],
       },
       Color = {
-        [[(<[a-zA-Z_,:]+>)]],
-        rex.new [[<([a-zA-Z_]+)?(?:[:,](?=>))?(?:[:,]([a-zA-Z_]+))?>]],
+        [[(<[a-zA-Z0-9_,:]+>)]],
+        rex.new [[<([a-zA-Z0-9_]+)?(?:[:,](?=>))?(?:[:,]([a-zA-Z0-9_]+))?>]],
```

Comment:
:+1:

Reply:
That had me scratching my head for an hour or more...!